front/components/assistant/conversation/MentionValidationRequired.tsx

```diff
@@ -80,12 +113,12 @@ export function MentionValidationRequired({
               </span>{" "}
               mentioned
               <span className="font-semibold">{pendingMention.label}</span>. Do
-              you want to invite them? They’ll see the full history and be able
+              you want to invite them? They'll see the full history and be able
```

Comment:
is it formatter or you updated it intentionally? (because I intentionally put the curly one `’`)